pymc_marketing/mmm/tvp.py

```diff
@@ -17,20 +17,94 @@
 import pymc as pm
 import pytensor.tensor as pt
 
-from pymc_marketing.constants import DAYS_IN_YEAR
+
+def pc_prior_1d(name, alpha=0.1, lower=1.0):
+    """
+    One dimensional PC prior for GP lengthscales, parameterized by tail probability:
+    p(lengthscale < lower) = alpha.
```

Comment:
This looks very nice! Would it be possible to get a bit more info about the nature of this prior and the benefits ? This can help us (and our users) whoa re not familiar with this :)